can-observation_test.js

```diff
@@ -430,7 +430,7 @@ QUnit.test("Observation can listen to something decorated with onValue and offVa
 });
 
 
-QUnit.test("Observation can listen to something decorated with onValue and offValue", function(){
+QUnit.test("Observation can listen to observable decorated with onValue and offValue", function(){
```

Comment:
This was named the same thing as the test above, so I gave them more distinct names.